2.0.0/arm64/Dockerfile

```diff
@@ -45,6 +45,7 @@ RUN apt-get update && \
       netbase \
       unzip \
       wget \
+      fontconfig:arm64 \
```

Comment:
You can safley remove the :arm64 or :armhf suffix from this package, because these images are already on the given architecture.